### PR TITLE
Added .js to project. Tailwind css used for styling and compiled. Add…

### PR DESCRIPTION
**Pull request by DeusCorona**
…ed importmaps. Root was fully styled except for contacts form.

### CODE REVIEW

File: app/javascript/custom/kindergarten.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+var toggle = true;
```
Comment (duleorlovic) [inline app/javascript/custom/kindergarten.js:1]: Can we use Stimulus instead event listeners ?

File: app/views/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "bg-white text-blue-500 rounded-md p-4 text-sm w-full" %>
```
Comment (duleorlovic) [inline app/views/devise/passwords/new.html.erb:10]: Do we use auto sort ?
This https://tailwindcss.com/blog/automatic-class-sorting-with-prettier
should be added to linter as well https://github.com/trkin/kindergarten-exchange/issues/11

**Review by duleorlovic — CHANGES_REQUESTED** [review]

File: app/javascript/custom/kindergarten.js (hunk 1 above)
Comment (DeusCorona) [reply app/javascript/custom/kindergarten.js:1]: I can try. But keep in mind that there's a bug in Tailwind when dynamically changing screensize which affects breakpoint classes eg. changing from low to high res dinamically can cause navbar menu to break down (changing from mobile to 768px causes navbar to remain hidden at certain conditions). This is why custom .js is unavoidable.

**Comment by duleorlovic** [comment]
@DeusCorona please use i123_my_branch naming https://dev.trk.in.rs/2023/03/21/git-tips.html

File: app/javascript/custom/kindergarten.js (hunk 1 above)
Comment (duleorlovic) [reply app/javascript/custom/kindergarten.js:1]: @DeusCorona please create another issue for that bug so we can discuss there

**Pull request closed without merging**